src/main/webapp/app/entities/student-with-team.model.ts

```diff
@@ -0,0 +1,6 @@
+export class StudentWithTeam {
+    public Name: string;
+    public Surname: string;
+    public 'Registration Number': string;
+    public 'Team Name': string;
```

Comment:
is this notation intended?

Reply:
Yes, since the requested JSON on issue has that notation, it automatically parses it. And throws error if something is missing.

Reply:
I think we should change these property names because it was just an example given by the issue reporter. 

He/she mentioned at the bottom that Moodle exports the following information:

```
Nachname;Vorname;Matrikelnummer;E-Mail-Adresse;Gruppe
Doe;John;01234567,john.doe@example.com,Team 1
Doe;Alice;01234568,alice.doe@example.com,Team 1
Doe;Bob;01234569,bob.doe@example.com,Team 2
```

So "Team Name" is not something that already exists and I'd just change that to `teamName` or `team`. Same thing goes for "Registration Number". Since the Matrikelnummer is the number that is unique between students, wouldn't it make sense to use that instead? It's just a question since I'm not sure.

Reply:
Matrikelnummer is German though. I am closing this pull request since I opened a new one with an internal branch. However, I am going to ask this question on slack.